client/components/domains/register-domain-step/index.jsx

```diff
@@ -1406,13 +1413,14 @@ class RegisterDomainStep extends Component {
 						this.showAvailabilityErrorMessage( domain, status, {
 							availabilityPreCheck: true,
 						} );
+						this.props.onMappingError( domain, status );
```

Comment:
Do we need to call this for `trademarkClaimsNoticeInfo` too? That would be a similar behavior code-for-code

Reply:
Also a good catch! I investigated it here: p1700172102072579-slack-C0BNMNMNG

[After this commit](https://github.com/Automattic/wp-calypso/pull/84171/commits/98a320e12adaffdd2d33f39a4bcf2b7bd7999703), this branch is working as same as trunk. I created a follow-up task (4548-gh-Automattic/dotcom-forge) since both branchs are partially broken.

Reply:
Also a good catch! I investigated it here: p1700172102072579-slack-C0BNMNMNG

[After this commit](https://github.com/Automattic/wp-calypso/pull/84171/commits/98a320e12adaffdd2d33f39a4bcf2b7bd7999703), this branch is working as same as trunk. I created a follow-up task (4548-gh-Automattic/dotcom-forge) since both branches are partially broken.